### PR TITLE
Add "Learn about recovery phrase" step from "Create Wallet" flow

### PR DESCRIPTION
**Pull request by neuodev**
Screen Shot 2023-02-25 at 11 45 07 PM

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[Create wallet steps components wrapper](https://emurgo.atlassian.net/browse/YOEXT-529)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/app/components/wallet/create-wallet/CreateWalletSteps.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    return (
+      <Box>
+        <Stack direction='row' alignItems='center' justifyContent='center' gap='24px' mt='24px' mb='48px'>
+          {steps.map(([stepId, label]) => {
+            const stepColor = currentStep === stepId ? 'primary.200' : 'grey.400';
+            return (
+              <Stack direction='row' alignItems='center' justifyContent='center' key={stepId}>
+                <Box
+                  sx={{
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/create-wallet/CreateWalletSteps.js]: We should use DS tokens

File: packages/yoroi-extension/app/components/wallet/create-wallet/CreateWalletSteps.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    const steps = [
+        [1, messages.firstStep],
+        [2, messages.secondStep],
+        [3, messages.thirdStep],
+        [4, messages.forthStep],
+    ];
+
+    return (
+      <Box>
+        <Stack direction='row' alignItems='center' justifyContent='center' gap='24px' mt='24px' mb='48px'>
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/create-wallet/CreateWalletSteps.js]: Create a DS component so we can reuse it.

File: packages/yoroi-extension/app/components/wallet/create-wallet/StepController.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+type Intl = {|
+  intl: $npm$ReactIntl$IntlShape,
+|};
+
+type Props = {|
+    goNext(): void,
+    goBack(): void,
+|};
+
+function LearnAboutRecoveryPhrase(props: Props & Intl): Node {
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/create-wallet/StepController.js]:
```suggestion
function StepController(props: Props & Intl): Node {
```

File: packages/yoroi-extension/app/components/wallet/create-wallet/StepController.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    const { intl, goBack, goNext } = props;
+
+    return (
+      <Stack direction='row' alignItems='center' justifyContent='center' mt='14px' py='24px' gap='24px'>
+        <Button
+          variant='outlined'
+          disableRipple={false}
+          onClick={goBack}
+          sx={{
+            width: '144px',
```
Comment (nistadev) [inline packages/yoroi-extension/app/components/wallet/create-wallet/StepController.js:34]: Use DS Tokens

**Review by zuzunker — APPROVED** [review]

**Pull request merged**